quickwit-metastore/src/metastore.rs

```diff
@@ -39,13 +39,16 @@ pub type IndexUri = String;
 /// A split ID.
 pub type SplitId = String;
 
+/// A file format version.
 pub static FILE_FORMAT_VERSION: &str = "0";
 
+/// An index metadata carries all meta data about a split.
```

Comment:
i see no information about splits in this struct what am i missing?

Reply:
Sorry, I made a mistake.